language/move-bytecode-verifier/transactional-tests/tests/dependencies/call_integers_valid.move

```diff
@@ -0,0 +1,22 @@
+//# publish
+module 0x42::m {
+public fun plus8(a: u8): u8 { a + 1 }
+public fun plus16(a: u16): u16 { a + 1 }
+public fun plus32(a: u32): u32 { a + 1 }
+public fun plus64(a: u64): u64 { a + 1 }
+public fun plus128(a: u128): u128 { a + 1 }
+public fun plus256(a: u256): u256 { a + 1 }
```

Comment:
formatting issue here

Reply:
It's just a test :) This is setup this way to make it easy to see each case

While the lack of newline after `{` is a bit obtuse. I've come to like not indenting inside of the `module` block. Not really a point in doing so IMO